src/modules/popUp.js

```diff
@@ -0,0 +1,32 @@
+import displayComments from './displayComments.js';
+import addComment from './addComment.js';
+
+const addEventToCommentBtn = () => {
+  const getAllBtn = document.querySelectorAll('.commentBtn');
+  getAllBtn.forEach((item) => {
+    // eslint-disable-next-line func-names
```

Comment:
I suggest you avoid disabling linter errors instead slove them for greater code quality